File: testcases/x86_testcases/x86_output/test6/test6.c
#include <stdio.h>

// Function with more than 6 parameters
int calculate_sum(int a, int b, int c, int d, int e, int f, int g) {
    return a + b + c + d + e + f + g;
}

void main2() {
    // Array assignments and indexing
    int array[7] = {1, 2, 3, 4, 5, 6, 7};
    printf("The value of 3rd number in array is: ");
    printf("%d", array[2]);
    printf("\n");
    array[2] = 10;
    printf("New value of 3rd number in array is: ");
    printf("%d", array[2]);
    printf("\n");

    // Call to function with more than 6 parameters
    int sum_fixed = calculate_sum(0, 1, 2, 3, 4, 5, 6);
    printf("Sum of first 7 whole numbers is: ");
    printf("%d", sum_fixed);
    printf("\n");

    // Call to function with more than 6 parameters
    int sum = calculate_sum(array[0], array[1], array[2], array[3], array[4], array[5], array[6]);
    printf("Sum of array is: ");
    printf("%d", sum);
    printf("\n");

    return;
}

int main() {
    main2();
    return 0;
}

File: testcases/x86_testcases/test6.py
# Function with more than 6 parameters
def calculate_sum(a: int, b: int, c: int, d: int, e: int, f: int, g: int) -> int:
    return a + b + c + d + e + f + g

# Array assignments and star expressions
def main() -> None:
    # Array operations using indexing
    array : list[int] = [1, 2, 3, 4, 5, 6, 7]
    print("The value of 3rd number in array is: ", array[2])
    array[2] = 10
    print("New value of 3rd number in array is: ", array[2])

    # Call to function with more than 6 parameters
    sum_fixed : int = calculate_sum(0, 1, 2, 3, 4, 5, 6)
    print("Sum of first 7 whole numbers is: ", sum_fixed)
    
    # Call to function with more than 6 parameters
    sum : int = calculate_sum(array[0], array[1], array[2], array[3], array[4], array[5], array[6])
    print("Sum of array is: ", sum)

    return

if __name__ == "__main__":
    main()
